web/src/lib/components/album-page/__tests__/album-cover.spec.ts

```diff
@@ -0,0 +1,49 @@
+import AlbumCover from '$lib/components/album-page/album-cover.svelte';
+import { getAssetThumbnailUrl } from '$lib/utils';
+import type { AlbumResponseDto } from '@immich/sdk';
+import { render } from '@testing-library/svelte';
+import { init, register, waitLocale } from 'svelte-i18n';
+
+vi.mock('$lib/utils');
+
+describe('AlbumCover component', () => {
+  beforeAll(async () => {
+    await init({ fallbackLocale: 'en-US' });
+    register('en-US', () => import('$lib/i18n/en-US.json'));
+    await waitLocale('en-US');
```

Comment:
I can totally see how having a test that tests against the translation gives you additional confidence in the code, but with the way translations are committed to the repo, it would be a pain to ensure translations don't require test changes too. I'll keep it as `en-US` just without loading the translations as that's what other tests do, though I'm happy to change it to something else.

Reply:
You can just use `'dev'` https://github.com/immich-app/immich/blob/0f976edf96af551731ca689fea269fc7ada6a632/web/src/lib/constants.ts#L254